packages/react-app/src/views/BuilderListView.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useEffect, useState} from "react"
+import { List, Typography } from "antd";
+import axios from "axios";
+import BuilderCard from "../components/BuilderCard";
+
+const { Title } = Typography;
+const serverPath = "builders";
+
+export default function BuilderListView({ serverUrl, mainnetProvider }) {
```

Comment:
Regarding `mainnetProvider`, I'm not sure when we'lI need it. I think it's fine to send it to the BuilderListView to just pass it to the BuilderCard. However, if we think we'll end up using it in many other places we can create a context like `MainnetProviderContext`.

A context has its drawbacks, like we'll always need to have the Address component wrapped in a `MainnetProviderContext`, creating an implicit dependency.

This is not something to change now at all. I'm just mentioning to keep it in mind if we see ourselves passing `mainnetProvider` around too often. And providers are usually good candidates to eventually put in a context, when the need arises.

Reply:
We use `mainnetProvider` it to resolve addresses to ENS. You're right, we should consider `context` if we are going to do this "prop passing" often.